src/BaGet.Aws/S3StorageOptions.cs

```diff
@@ -22,5 +22,11 @@ public class S3StorageOptions
         public bool UseInstanceProfile { get; set; }
 
         public string AssumeRoleArn { get; set; }
+
+        public bool ForcePathStyle { get; set; } = false;
```

Comment:
ASP.NET Core will default this to false automatically, no need to be explicit here:

```suggestion
        public bool ForcePathStyle { get; set; }
```